outputs.tf

```
@@ -45,7 +45,7 @@ output "cluster_iam_role_arn" {
 
 output "cluster_oidc_issuer_url" {
   description = "The URL on the EKS cluster OIDC Issuer"
-  value       = concat(aws_eks_cluster.this[*].identity[*].oidc.0.issuer, [""])[0]
+  value       = concat(aws_eks_cluster.this[*].identity.0.oidc.0.issuer, [""])[0]
```

This will break on clusters that were created/upgraded before September 3rd or still running 1.12. They don't have an identity provider and terraform will return an error.

Maybe throwing in a  `flatten` will resolve the problem for both parties?
```suggestion
  value       = flatten(concat(aws_eks_cluster.this[*].identity[*].oidc.0.issuer, [""]))[0]
```

```hcl
output "a" {
  value = flatten(concat([["a"]], [""]))[0]
} 

output "b" {
  value = flatten(concat([], [""]))[0]
}
```
```text
Outputs:

a = a
b =
```